clients/client-test/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
this was added to keep `clients/client` clean, and separate tests that used internal libraries from the client itself.